src/WebJobs.Script.WebHost/applicationHost.xdt

```diff
@@ -4,7 +4,7 @@
     <sites>
       <site name="%XDT_SITENAME%" xdt:Locator="Match(name)">
         <application path="/" xdt:Locator="Match(path)" xdt:Transform="Remove" />
-        <application path="/" applicationPool="%XDT_APPPOOLNAME%" preloadEnabled="true" xdt:Transform="Insert">
```

Comment:
This change was done for our warm site work (in this commit https://github.com/Azure/azure-webjobs-sdk-script/commit/51b31a7f02bc33442d13b67420a550613143f62a#diff-0e5dedfbc79c5f4b37a86d48dccb27dbR7), which isn't landing for GA so we're reverting this for now.